### PR TITLE
[WIP] Add docusaurus

### PR DESCRIPTION
**Pull request by sasandhu**
Update docs to use docusaurus. Added a docs-new folder and moved the existing md files to this folder

Landing page
image

Guide
image

### CODE REVIEW

**Comment by sasandhu** [comment]
Yet to update the icons and playing around with the colors as well

File: .github/workflows/docusaurus-release.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          yarn build
+
+      # Popular action to deploy to GitHub Pages:
+      # Docs: https://github.com/peaceiris/actions-gh-pages#%EF%B8%8F-docusaurus
+      - name: Deploy to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          # Build output to publish to the `gh-pages` branch:
+          publish_dir: ./docs-new/build
```
Comment (kenotron) [inline .github/workflows/docusaurus-release.yml:31]: todo: add destination_dir like this:
https://github.com/peaceiris/actions-gh-pages#%EF%B8%8F-deploy-to-subdirectory-destination_dir

File: docs-new/docusaurus.config.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+const lightCodeTheme = require('prism-react-renderer/themes/github');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: 'Lage',
+  tagline: 'A Beautiful JS Monorepo Task Runner',
+  url: 'https://microsoft.github.io',
+  baseUrl: '/',
```
Comment (kenotron) [inline docs-new/docusaurus.config.js:12]: pretty sure this has to be lage/docs-new

Comment (sasandhu) [reply docs-new/docusaurus.config.js]: updated the url to -` https://microsoft.github.io/lage` and baseUrl to - `/docs-new`

**Review by kenotron — APPROVED** [review]

**Pull request merged**